src/tests/ci.md

```diff
@@ -40,35 +40,31 @@ For example, if a Windows build fails, but you don't have access to a Windows
 machine, you can try running the Windows job that failed on CI within your PR
 after pushing a possible fix.
 
-To do this, you'll need to edit [`src/ci/github-actions/ci.yml`].
-The `jobs` section defines the jobs that will run.
-The `jobs.pr` section defines everything that will run in a push to a PR.
-The `jobs.auto` section defines the full set of tests that are run after a PR is approved.
-You can copy one of the definitions from the `auto` section up to the `pr` section.
+To do this, you'll need to edit [`src/ci/github-actions/jobs.yml`]. It contains three
+sections that affect which CI jobs will be executed:
+- The `pr` section defines everything that will run after a push to a PR.
+- The `try` section defines job(s) that are run when you ask for a try build using `@bors try`. 
+- The `auto` section defines the full set of tests that are run after a PR is approved and before
+it is merged into the main branch.
 
-For example, the `x86_64-msvc-1` and `x86_64-msvc-2` jobs are responsible for
-running the 64-bit MSVC tests.
-You can copy those up to the `jobs.pr.strategy.matrix.include` section with
-the other jobs.
+You can copy one of the definitions from the `auto` section to the `pr` or `try` sections.
+For example, the `x86_64-msvc` job is responsible for running the 64-bit MSVC tests.
+You can copy it to the `pr` section to cause it to be executed after a commit is pushed to your
+PR.
```

Comment:
I think this should mention that you should not copy the YAML anchors in `env` if the job description has it.

Reply:
Also could you add an example for adding a job to PR CI (where the job description has a YAML anchor, such as one of the apple jobs)? I think that would be very helpful for people trying to edit PR CI to test stuff.

Reply:
We should just remove these anchors altogether, I'll send a PR to `rust`.

Reply:
I added an example how to add a job to PR CI. Hopefully this will soon be needed less once we get arbitrary try builds.

Reply:
Thanks for the improvements 💚